apiary.apib

```diff
@@ -543,5 +560,12 @@ Represents a system of record user.
 
 ### Properties
 + `userName`: soruser123 (string, required) - The user name for the system of record user.
-+ `sourceAbbreviation`: SD (string, required) - The the source abbreviation for the system of record(like SD for School Dude or FD for Facility Dude).
++ `sourceAbbreviation`: SD (string, required) - The source abbreviation for the system of record(like SD for School Dude or FD for Facility Dude).
 + `userSummaryId`: 1 (number, required) - {INT} The dude platform user.
+
+## System Of Record ApiClient (Api Client Audit)
+Represents a mapping between an Api Client and a System of Record.
+
+### Properties
++ `apiClientIdentifier`: abc123 (string, required) - The client identifier for the system of record.
```

Comment:
We used MyClientName in the method definition but abc123 in the data structure. We should probably change the data structure to match.